src/Streamly/Tutorial.hs

```diff
@@ -244,20 +250,20 @@ import Control.Monad.Trans.Class   (MonadTrans (lift))
 -- @
 --
 -- All these types can be freely inter-converted using type conversion
--- combinators or type annotations without any cost, to acheive the desired
--- composition style.  To force a particular type of composition we coerce the
+-- combinators or type annotations, without any cost, to achieve the desired
+-- composition style.  To force a particular type of composition, we coerce the
 -- stream type using the corresponding type adapting combinator from
 -- 'serially', 'aheadly', 'asyncly', or 'parallely'.  The default stream type
 -- is inferred as 'Serial' unless you change it by using one of the combinators
--- or using a type annotation.
+-- or by using a type annotation.
 
 -- $flavors
 --
--- Streams can be combined using semigroup or monoid composition to form a
--- composite stream. Traversal of a composition of streams could be @deep@ or
--- @wide@.  Deep goes depth first i.e.  each stream is traversed fully before
--- we traverse the next stream. Wide goes breadth first i.e. one element from
--- each stream is traversed before coming back to the first stream again.
+-- Streams can be combined using '<>' or 'mappend' to form a composite
+-- stream. Deep streams are combined depth first i.e. each stream is traversed
```

Comment:
It may appear that we are "combining" streams depth first or breadth first. But in fact combining does not matter because we can always interconvert the types from deep to wide or serial to parallel before we consume it. Therefore a better way of expressing the idea would be to explain consumption or traversal rather than combining because that is when it is decided whether we are traversing it depth first or breadth first.

So can we say something like this:

"Composite streams can be interpreted in a depth first or breadth first manner using an appropriate type conversion before consumption. Deep (e.g. 'Serial') stream type variants traverse a composite stream in a depth first manner whereas wide (e.g. 'WSerial') stream types traverse it in a breadth first manner".